apps/site/components/Navigation/NavbarLinksUsage.tsx

```diff
@@ -17,7 +17,7 @@ function NavbarLinksUsage({ classes }: { classes?: string }) {
 
   return (
     <NavbarLinks className={`${classes}`}>
```

Comment:
This is not one change from this PR but is a quick fix.
```suggestion
    <NavbarLinks className={classes}>
```